CONTRIBUTING.md

```diff
@@ -32,12 +32,18 @@ However, this project supports Bazel build environments as well. As such, the Ba
 to be kept up-to-date.
 
 When new files are added, or existing ones are deleted or moved, the appropriate BUILD files must
-be updated. This can, and should, be done automatically via Gazelle e.g. `bazel run //:gazelle`.
+be updated. This can, and should, be done automatically via Gazelle e.g. `make gazelle`.
+
+_Note: If running on linux, you must comment remove the `''` value of the `-i` flag in the this
+target's `sed` command._
 
 If there are changes to the `go.mod`, an equivalent change must be made on the `repositories.bzl`
 macro that defines the Go repository dependencies. This can be done manually or by executing the
 Make target: `make update-bazel-repos`.
 
+_Note: If running on linux, you must comment remove the `''` value of the `-i` flag in the this
+target's `sed` command._
+
```

Comment:
I know this is silly, but I can't be bothered to make a target that checks the OS and changes how `sed` is invoked.